crates/epoch-snark/src/api/prover.rs

```diff
@@ -24,6 +22,7 @@ pub fn prove(
     num_validators: u32,
     initial_epoch: &EpochBlock,
     transitions: &[EpochTransition],
+    max_transitions: usize,
```

Comment:
The idea is that the consumer of this will make it constant. The consumer is a program pulling blocks from the Celo mainnet and feeding this into this function.

Reply:
Isn't `max_transitions` determined by a single circuit for a fixed number of epoch messages we'll be doing the setup for? Isn't `transitions` what will vary based on how many epochs they are actually proving?

Reply:
Yep, max transitions is determined by the setup we'll be making. The question is where we put the parameters of the setup. Currently it's not here.